maker/src/metrics.rs

```diff
@@ -0,0 +1,152 @@
+use lazy_static::lazy_static;
+use lightning::ln::channelmanager::ChannelDetails;
+use ln_dlc_node::node::InMemoryStore;
+use ln_dlc_node::node::Node;
+use opentelemetry::global;
+use opentelemetry::metrics::Meter;
+use opentelemetry::metrics::ObservableGauge;
+use opentelemetry::sdk::export::metrics::aggregation;
+use opentelemetry::sdk::metrics::controllers;
+use opentelemetry::sdk::metrics::processors;
+use opentelemetry::sdk::metrics::selectors;
+use opentelemetry::Context;
+use opentelemetry::KeyValue;
+use opentelemetry_prometheus::PrometheusExporter;
+use std::sync::Arc;
+use std::time::Duration;
+
+lazy_static! {
+    pub static ref METER: Meter = global::meter("coordinator");
```

Comment:
I assume this should be `maker`.
```suggestion
    pub static ref METER: Meter = global::meter("maker");
```